utils/lib/mux.py

```diff
@@ -123,19 +123,19 @@ def pb_type_xml(mux_type, mux_name, pins, subckt=None, num_pb=1, comment=""):
 
     if mux_type == MuxType.LOGIC:
         if '-' not in mux_name:
-            mux_name = 'BEL_MX-' + mux_name
+            mux_name = '' + mux_name
         else:
             assert mux_name.startswith(
-                'BEL_MX-'
+                ''
```

Comment:
does this assertion make sense now that prefixes from the names are gone?

Reply:
Nope!